File: (8 kyu) Counting sheep/(8 kyu) Counting sheep.cs
using System;

public static class Kata
{
  public static int CountSheeps(bool[] sheeps)
  {
    int res = 0;
    for (int i = 0; i < sheeps.Length; i++)
    {
      if (sheeps[i] == true)
      {
        res++;
      }
    }
    return res;
  }
}

File: (8 kyu) Counting sheep/(8 kyu) Counting sheep.go
package kata

func CountSheeps(arrayOfSheep []bool) int {
  var res int
  
  for i := 0; i < len(arrayOfSheep); i++ {
    if arrayOfSheep[i] == true {
      res += 1
    }
  }
  
  return res
}
